Archivist.lua

```diff
@@ -255,6 +261,33 @@ function proto:RegisterStoreType(prototype)
 	end
 end
 
+-- produces storable data from image
+function proto:Wind(storeType, image)
+	do -- arg validation
+		self:Assert(type(storeType) == "string" and self.prototypes[storeType], "Store type must be registered before loading data.")
+		self:Assert(image ~= nil, "Cannot wind nil data")
+	end
+
+	if self.prototypes[storeType].Wind then
+		return self.prototypes[storeType]:Wind(image)
+	else
+		return self:Archive(image)
+	end
+end
+
+function proto:Unwind(storeType, woundImage)
+	do -- arg validation
+		self:Assert(type(storeType) == "string" and self.prototypes[storeType], "Store type must be registered before loading data.")
```

Comment:
Both Wind and Unwind say "before loading data" in their assertion message; is this just a copy/paste from `proto:Create()` or intended?

Reply:
oops yeah it is